domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -745,6 +745,88 @@ private static void streamTrailer(HttpServerResponse response, ResultInfo result
     }
   }
 
+   /**
+   * Delete records by CQL.
+   * @param table  the table that contains the records
+   * @param cql  the CQL query for filtering and sorting the records
+   * @param offset number of records to skip, use 0 or negative number for not skipping
+   * @param limit maximum number of records to return, use a negative number for no limit
+   * @param okapiHeaders  http headers provided by okapi
+   * @param vertxContext  the current context
+   * @param responseDelegateClass  the ResponseDelegate class generated as defined by the RAML file,
+   *    must have these methods: respond200(C), respond400WithTextPlain(Object), respond500WithTextPlain(Object).
```

Comment:
Can you explain in the javadoc what content the JSON will have that is returned along with the HTTP 200 status code on success?
If we don't want to return anything we can use the 204 status code.